src/core/js/views/drawerView.js

```diff
@@ -86,7 +86,14 @@ define([
             this._isCustomViewVisible = true;
             Adapt.trigger('drawer:empty');
             this.showDrawer();
+
             this.$('.drawer-holder').html(view);
+
+            var model = this.collection.findWhere({
```

Comment:
Hi @sarveshwar-gavhane, I tried it with pageLevelProgress but did not see any problems. Could you perhaps give the steps to reproduce any problems you've seen?